src/release-notes/v2022.02.17-0.md

```diff
@@ -0,0 +1,14 @@
+---
+date: "2022-02-17"
+version: "v2022.02.17-0"
+weight: 202202170
+---
+
+### <span class="label label-green">New Features</span>
+- The kotsadm addon no longer includes the minio image.
+
+### <span class="label label-orange">Bug Fixes</span>
+- Fix an erroneous host preflight failure when using EKCO's [internal load balancer](https://kurl.sh/docs/add-ons/ekco#internal-load-balancer).
+- Fix that detects host's private IP automatically instead of asking via prompt.
```

Comment:
I don't follow this note exactly. Can it be explained a bit more? Like when is this prompted or what part of the UX is this?

Reply:
Reworded to add details on where in UX this change is seen.  Also moved it to an improvement.